BrownianMotion/Gaussian/BrownianMotion.lean

```diff
@@ -114,41 +114,118 @@ lemma memHolder_brownian (ω : ℝ≥0 → ℝ) (t : ℝ≥0) (β : ℝ≥0) (h
+    rw [Measure.map_map (by fun_prop) (measurable_pi_lambda _ measurable_brownian)]
+    exact hasLaw_restrict_brownian.map_eq
+
+lemma hasLaw_brownian_eval {t : ℝ≥0} :
+    HasLaw (brownian t) gaussianLimit (gaussianReal 0 t) :=
+  (hasLaw_preBrownian_eval t).congr (brownian_ae_eq_preBrownian t)
+
+lemma hasLaw_brownian_sub {s t : ℝ≥0} :
+    HasLaw (brownian s - brownian t) gaussianLimit (gaussianReal 0 (max (s - t) (t - s))) :=
+  (hasLaw_preBrownian_sub s t).congr
+    ((brownian_ae_eq_preBrownian s).sub (brownian_ae_eq_preBrownian t))
+
+instance isGaussianProcess_brownian : IsGaussianProcess brownian gaussianLimit :=
+  isGaussianProcess_preBrownian.modification fun t ↦ (brownian_ae_eq_preBrownian t).symm
 
 open NNReal Filter Topology in
 lemma measurable_brownian_uncurry : Measurable brownian.uncurry :=
   measurable_uncurry_of_continuous_of_measurable continuous_brownian measurable_brownian
 
-lemma isGaussianProcess_brownian : IsGaussianProcess brownian gaussianLimit :=
-  isGaussianProcess_preBrownian.modification fun t ↦ (brownian_ae_eq_preBrownian t).symm
-
 lemma isMeasurableKolmogorovProcess_brownian (n : ℕ) :
     IsMeasurableKolmogorovProcess brownian gaussianLimit (2 * n) n
       (Nat.doubleFactorial (2 * n - 1)) where
   measurablePair := measurable_pair_of_measurable measurable_brownian
   kolmogorovCondition := (isMeasurableKolmogorovProcess_preBrownian n).isKolmogorovProcess.congr
     (fun t ↦ (brownian_ae_eq_preBrownian t).symm) |>.kolmogorovCondition
 
+lemma iIndepFun_iff_charFun_eq_pi {ι Ω : Type*} [Fintype ι] {E : ι → Type*}
+    [∀ i, NormedAddCommGroup (E i)]
+    [∀ i, InnerProductSpace ℝ (E i)] [∀ i, MeasurableSpace (E i)]
+    {mΩ : MeasurableSpace Ω} {μ : Measure Ω} [IsProbabilityMeasure μ] {X : Π i, Ω → (E i)}
+    [∀ i, CompleteSpace (E i)] [∀ i, BorelSpace (E i)]
+    [∀ i, SecondCountableTopology (E i)] (mX : ∀ i, AEMeasurable (X i) μ) :
+    iIndepFun X μ ↔ ∀ t,
+      charFun (μ.map fun ω ↦ toLp 2 (X · ω)) t = ∏ i, charFun (μ.map (X i)) (t i) := sorry
+-- PR #26269 in Mathlib
+
+lemma HasGaussianLaw.iIndepFun_of_covariance_eq_zero {ι Ω : Type*} [Fintype ι]
+    {mΩ : MeasurableSpace Ω} {P : Measure Ω} [IsProbabilityMeasure P] {X : ι → Ω → ℝ}
+    [h1 : HasGaussianLaw (fun ω ↦ (X · ω)) P] (h2 : ∀ i j : ι, i ≠ j → cov[X i, X j; P] = 0) :
+    iIndepFun X P := by
+  refine iIndepFun_iff_charFun_eq_pi (fun _ ↦ h1.aemeasurable.eval) |>.2 fun ξ ↦ ?_
+  simp_rw [HasGaussianLaw.charFun_toLp, ← sum_sub_distrib, Complex.exp_sum,
+    HasGaussianLaw.charFun_map_real]
+  congrm ∏ i, Complex.exp (_ - ?_)
+  rw [Fintype.sum_eq_single i]
+  · simp [covariance_self, h1.aemeasurable.eval, pow_two, mul_div_assoc]
+  · exact fun j hj ↦ by simp [h2 i j hj.symm]
+
+/-- A process `X : T → Ω → E` has independent increments if for any `n ≥ 2` and `t₁ ≤ ... ≤ tₙ`,
+the random variables `X t₂ - X t₁, ..., X tₙ - X tₙ₋₁` are independent. -/
+def HasIndependentIncrements {Ω T E : Type*} {mΩ : MeasurableSpace Ω} [Sub E]
```

Comment:
Every definition about independence in Mathlib uses Indep instead of Indepedent, so for uniformity we should also shorten this one
```suggestion
def HasIndepIncrements {Ω T E : Type*} {mΩ : MeasurableSpace Ω} [Sub E]
```